templates/paper.html

```diff
@@ -159,6 +159,53 @@ <h6 class="card-subtitle text-muted" align="center">
   })
 </script>
 
+<script src="static/js/add-to-calendar.js"></script>
 
+<script>
+  let ouicalData;
+  let targetNode;
+  let calendarNodeInner;
+  let calendarNodeSpan;
+  let parser = new DOMParser();
+  let calendarNames = Array("google", "off365", "outlook", "ical");
+
+  {% for session in paper.content.sessions %}
+  // create a calendar by hand
```

Comment:
I used the method 3 https://github.com/commonpike/add-to-calendar-buttons#method-3